projects/plugins/boost/app/modules/critical-css/class-critical-css.php

```diff
@@ -703,7 +698,7 @@ public function asynchronize_stylesheets( $html, $handle, $href, $media ) {
 		$html_no_script = '<noscript>' . $html . '</noscript>';
 
 		// Update the stylesheet markup for allowed methods.
-		$html = preg_replace( '~media=[\'"]?[^\'"\s]+[\'"]?~', $available_methods[ $method ], $html );
+		$html = preg_replace( '~media=[\'"]?([^"]*)[\'"]~', $available_methods[ $method ], $html );
```

Comment:
The regex can fail here on strings like `<link media='foo bar fighters'/><link media='foo bar fighters'/>`
![Screen Shot 2021-11-16 at 12 00 05 PM](https://user-images.githubusercontent.com/988095/141964436-fc791920-e918-4ed5-830b-e75752dbf36e.jpg)